ci/live-cluster-sanity.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+set -e
+cd "$(dirname "$0")/.."
+
+source ci/_
+source ci/rust-version.sh stable
+
+escaped_branch=$(echo "$BUILDKITE_BRANCH" | tr -c "[:alnum:]" - | sed -r "s#(^-*|-*head-*|-*$)##g")
+instance_prefix="testnet-live-sanity-$escaped_branch"
```

Comment:
I think metrics will complain about this since there won't be a database named `$instance_prefix`. (I hit similar trying to get cute with the rolling upgrades instance names)

Reply:
Fortunately, this pr doesn't use much of `net/*.sh`s. This isn't affected. Anyway, I've specifically setup a metric database for this job.